README.md

```diff
@@ -241,6 +241,37 @@ Linux or Mac using make
 These instructions are for building Simbody from source on either a Mac or on
 Ubuntu.
 
+#### Check the compiler version
+
+Simbody uses recent C++ features, that require a modern compiler.
+Before installing Simbody, check your compiler version with commands like that:
+
+- `g++ --version`
+- `clang --version`
+
+In case your compiler is not supported, you can upgrade your compiler.
+
+Here are some instructions to upgrade GCC on a Ubuntu 14.04 distribution.
+
+    $ sudo add-apt-repository ppa:ubuntu-toolchain-r/test
+    $ sudo apt-get update
+    $ sudo apt-get install gcc-4.9 g++-4.9
+
+If one wants to set `gcc-4.9` and `g++-4.9` as the default compilers, run the following command
+
+    $ sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-4.9 60 --slave /usr/bin/g++ g++ /usr/bin/g++-4.9
```

Comment:
After installing 4.9, it was not set as default compiler. I removed the old alternatives and added the 4.9, copying & pasting lines from random forums (adventurous life! :volcano:) and was able to use it, but maybe the `--config gcc` is sufficient. Perhaps tomorrow I can get my hands on a 14.04 virtual box and verify it, but I can't promise